tests/run-make/dump-ice-to-disk/check.sh

```diff
@@ -1,64 +0,0 @@
-#!/bin/sh
-
-# Default nightly behavior (write ICE to current directory)
```

Comment:
Question: AFAICT the ported test is not functionally equivalent to this shell script? Please forgive me if they *are* checked (but having comments in the ported test that corresponds to the sections in this shell script would help!

1. "Default nightly behavior (write ICE to current directory)" doesn't seem to be *explicitly* checked.
2. "Explicit directory set" doesn't seem to be *explicitly* checked. I don't see anothe `RUSTC_ICE=$TMPDIR`?

Reply:
> "Default nightly behavior (write ICE to current directory)" doesn't seem to be explicitly checked.

It's not checked in the test either, as these lines are commented out. I still tried to re-add them just now, but I don't see how `.` is different from `cwd()`. 

> "Explicit directory set" doesn't seem to be explicitly checked. I don't see another RUSTC_ICE=$TMPDIR 

Setting `RUSTC_ICE` to be `cwd()` changes nothing, I'm pretty sure, but I did it anyhow.